python/ql/src/semmle/python/regex.qll

```diff
@@ -620,7 +624,8 @@ abstract class RegexString extends Expr {
         exists(int y | this.lastPart(start, y) |
             this.emptyMatchAtEndGroup(end, y) or
             this.qualifiedItem(end, y, true) or
-            this.specialCharacter(end, y, "$")
+            this.specialCharacter(end, y, "$") or
+            y = end+2 and this.escapingChar(end) and this.getChar(end+1) = "Z"
```

Comment:
I'm not sure what problem you're seeing with this.

However, I can see that this predicate should be autoformatted. For a short while longer, please follow the instructions [here](https://github.com/github/codeql-python-team/issues/33#issue-569019344) to set up Python specific autoformatting that uses 4 spaces instead of 2.

Reply:
It came out with 2-space indent :-) But it was easy to modify the diff by hand..